credentials/src/test/kotlin/web5/sdk/credentials/PresentationExchangeTest.kt

```diff
@@ -0,0 +1,43 @@
+package web5.sdk.credentials
+
+import assertk.assertFailure
+import assertk.assertions.messageContains
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import org.junit.jupiter.api.assertDoesNotThrow
+import web5.sdk.crypto.InMemoryKeyManager
+import web5.sdk.dids.DidKey
+import kotlin.test.Test
+
+class PresentationExchangeTest {
+  private val keyManager = InMemoryKeyManager()
+  private val issuerDid = DidKey.create(keyManager)
+  private val holderDid = DidKey.create(keyManager)
+  private val jsonMapper: ObjectMapper = ObjectMapper()
+    .registerKotlinModule()
```

Comment:
Curious what this does?

Reply:
This one gets jackson's objectmapper to work nicely with kotlin

Reply:
Interesting! First time i've seen this. Thanks!